### DISCUSSION TITLE
many conmon "nwarn" messages in /var/log/messages

### DISCUSSION THREAD

**Discussion by jwcard** [containers/conmon#537] [Q&A]
I am using podman and crun (and of course conmon) in rootless mode (and cgroupsv2)
```
# conmon --version
conmon version 2.1.10
commit: d74eefa91394de92cb7ebb3d58651ab75d1c0662
```
I have noticed many thousands of nwarn messages in the current log. The applications work fine and these seem to be warnings.

I am assuming I can safely ignore these but would like to squelch these, if so either by addressing the issue or filtering them.

The messages are like these:
```
conmon ea855a74eaab734d2b3f <nwarn>: Failed to create inotify fd
conmon ea855a74eaab734d2b3f <nwarn>: Failed to write container stdin
conmon ea855a74eaab734d2b3f <nwarn>: Failed to write to remote console socket
```

**Comment by haircommander** [comment]
what log level are you using with podman?

**Reply by jwcard** [reply]
Just the default (I guess its WARN). I guess I can change that but I wasn't sure if I should be concerned about those messages.
